### DISCUSSION TITLE
Use AI to provide translations for documents

### DISCUSSION THREAD

**Discussion by hyzx86** [OrchardCMS/OrchardCore#13453] [General]
In last year, @agriffard  created a repository for Chinese document translation
https://github.com/OrchardCoreCn/OrchardCore

I translated some pages,
However, since English documents need to be updated frequently, human translation can't keep up with the response speed, and it is not worthwhile for users to waste time on incorrect or outdated documents.
So I stopped translating.

Could we use ChatGPT or another AI translation service to serve multilingual documents?

**Comment by hishamco** [comment]
Interesting, I was thinking about some translation connectors that might I include in OCC https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/issues/82

AI may be or may be not involved here, but at least it will save the time instead of doing them manually

**Comment by hyzx86** [comment]
@hishamco My idea is to use github's workflow implementation to listen to some markdown files or document directories and automatically translate and submit a PR when the document is updated

**Comment by hishamco** [comment]
> My idea is to use github's workflow implementation to listen to some markdown files or document directories and automatically translate and submit a PR when the document is updated

Think about how many PRs and more important which files will listened to, and how. Maybe it's better to trigger [PO Extractor](https://github.com/OrchardCoreContrib/OrchardCoreContrib.PoExtractor) on every PR, so whenever new localized string show up it will be added as a part of POT file

While POT files tracked in `OC.Translations` this may need some thinking

/cc @agriffard @sebastienros

**Comment by hyzx86** [comment]
Hi @agriffard ， I have traslated all of the docs  on here
https://github.com/OrchardCoreCn/OrchardCore/commit/14f0b22f769d3ca2899143fa32df7931415bcdd2

Can you help publish it? 
You may also need to update your repository Settings, such as setting main to be the main branch.
